libr/core/cmd_zign.c

```diff
@@ -731,9 +731,80 @@ struct ctxSearchCB {
 	const char *prefix;
 };
 
+static char *types_list_to_fcnstr(RList *types) {
+	char *type_kv = NULL, *k = NULL, *v = NULL;
+	char *field = NULL, *name = NULL, *rettype = NULL;
+	char **args, *ret = NULL;
```

Comment:
any reason not to use RList instead of char ** here? mmemory management and code readability will be better